data/mods/Aftershock/items/tool_quality.json

```diff
@@ -4,5 +4,10 @@
     "id": "CHURN",
     "name": "churn",
     "//": "Delete this when you mainline Dairy products existing PR"
```

Comment:
Please delete this quality since it exists in mainline now `"CHURN"`.   Just happened to notice my note to myself.

Reply:
Alright!
I'll do it in a separate pr tho.